### DISCUSSION TITLE
Can the Immich external library and the upload library from a phone be the same location?

### DISCUSSION THREAD

**Discussion by delicon** [immich-app/immich#11355] [Q&A]
This is my docker compose file:
```
volumes:
  - ${UPLOAD_LOCATION}:/usr/src/app/upload
  - ${THUMBS_LOCATION}:/usr/src/app/upload/thumbs
  - ${UPLOAD_LOCATION}/immich/upload:/usr/src/app/upload/upload
  - ${ENCODED_VIDEO_LOCATION}:/usr/src/app/upload/encoded-video
  - ${EXTERNAL_LOCATION}:/mnt/media
```

The .env file:
```
EXTERNAL_LOCATION=/mnt/media/Photos
UPLOAD_LOCATION=/mnt/media/Photos
THUMBS_LOCATION=/mnt/media/Photos/immich/thumbs
ENCODED_VIDEO_LOCATION=/mnt/media/Photos/immich/encoded-video
```

This is the directory structure:
```
/mnt/media/Photos
  - encoded-videos *
  - immich
    - encoded-videos
    - thumbs
    - upload
  - library
    - user1 library
    - user2 library
  - thumbs *
  - upload *

* these are automaticaly created by Immich but not used
```

**Like this, everything is working fine, but the images uploaded from the phone are shown twice.**

Here are the entries in the DB for the same photo:
```
upload/library/*NAME*/2024/07/IMG_7160.JPG
/mnt/media/library/*NAME*/2024/07/IMG_7160.JPG
```

Any suggestions what should I try to solve this problem?
Thanks

**Comment by bo0tzz** [comment]
No, they cannot. They should be separate, and the upload library wants an empty folder to start with.

**Reply by delicon** [reply]
So what should I do? I have a big external library, that I want to update it with the new photos we take with the phone. So all the photos will be in the same location separated by year/month.

**Reply by bo0tzz** [reply]
You could put the upload library in a subfolder of the external one, but more than that isn't possible with Immich.

**Reply by delicon** [reply]
Hey, my setup works. The photos are uploading in the same folder as the external library.
The solution for the duplicate photos was to disable the periodic scanning for the external library.
